2025REDOBuildSeason/src/main/java/frc/robot/Subsystem/Mecanum.java

```diff
@@ -95,5 +153,10 @@ public boolean isEnabled() {
     public String getName() {
         return "mecanum";
     }
-
+    public boolean moving() {
+        if(frontRight >= PortMap.MOVING_THRESHOLD || frontLeft >= PortMap.MOVING_THRESHOLD || rearLeft >= PortMap.MOVING_THRESHOLD || rearRight >= PortMap.MOVING_THRESHOLD) {
```

Comment:
```suggestion
        return frontRight >= PortMap.MOVING_THRESHOLD || frontLeft >= PortMap.MOVING_THRESHOLD || rearLeft >= PortMap.MOVING_THRESHOLD || rearRight >= PortMap.MOVING_THRESHOLD;
```

And remove code below